v2/tools/generator/internal/codegen/pipeline/testdata/Test_RenameProperties_WhenFlattening_PopulatesExpectedARMProperty/person-v20200101.golden

```diff
@@ -136,6 +136,16 @@
  
  const APIVersion_apiVersion = APIVersion(2020-06-01)
  
+ // +kubebuilder:validation:Enum={2020-06-01}
```

Comment:
minor: If you wanted you could strip this Kubebuilder validation because it doesn't do anything for teh ARM enums

Reply:
I'll make a note to clean this up in a later PR.